src/pages/resultsView/download/CaseAlterationTable.tsx

```diff
@@ -225,26 +239,30 @@ export default class CaseAlterationTable extends React.Component<ICaseAlteration
                     getPseudoOqlSummaryByAlterationTypes(data.oqlDataByGene, oql.gene, alterationTypes)!.summaryContent.toUpperCase().includes(filterStringUpper),
                 visible: false
             });
-            //add column for each gene alteration combination
-            alterationTypes.forEach(alterationType => {
-                columns.push({
-                    name: `${oql.gene} ${alterationType}`,
-                    tooltip: <span>{oql.oql_line}</span>,
-                    headerDownload: (name: string) => `${oql.gene} ${alterationType}`,
-                    render: (data: ICaseAlteration) => {
-                        const pseudoOqlSummary = generatePseudoOqlSummary(data.oqlDataByGene, oql.gene, alterationType); 
+        });
+
+        this.props.oqls.forEach(oql => {
+            //add column for each oql
```

Comment:
it hink this comment belongs above the forEach

Reply:
Right, I can merge them together.